### ISSUE TITLE
Data format version

### ISSUE THREAD

**Issue by keesnieuwstad** [mobilityDCAT-AP/mobilityDCAT-AP#3]
The **data format version** (mobilitydcatap:dataFormatVersion) is modelled as a rdfs:Literal. In the example it is filled with "DATEX II v3.2". Using Literal may make it difficult to filter the values entered by the various member states, because it also can be filled with, for example "DATEX-II-v3.2" or "DATEXIIv3.2" or even different values.

To overcome this isse, it may be an idea to use a controlled vacabulary for the data format version.

For example the EU Vocabularies include a controlled vocabulary for [File type](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/file-type).

A solution option could be to add, for example, DATEX_II_v3_XML and DATEXII_v2_3_XML and other for mobilityDCAT-AP needed filetypes to the File type defined in the EU vocabularies controlled vocabulary and use the SKOS concept instead of the Literal.

Of course, a precondition is that the EU vocabulary for filetype is actively managed and can be adapted quickly enough if filetypes need to be added for mobilityDCAT-AP. Otherwise, maybe an mobilityDCAT-AP controlled vocabulary may be used.

**Comment by peterlubrich** [comment]
We checked two approaches to resolve this issue:

1. Include the data model and its versions (e.g. "DATEX II v3.2") as additional values in the EU Controlled Vocabulary "File type": this Vocabulary is listing common syntax formats (e.g. XML), whereas DATEX II is more than that: it combines syntax and semantic elements of a domain model  (here road traffic information) and even adds technical rules for data exchange.  Thus, we decided to add an own property "mobilitydcatap:dataModel" where we list common data models from the mobility sector (via an own Controlled Vocabulary with values "DATEX II", "TN-ITS", "NeTEx" etc.)
2.  To include the version number as a suffix to the values of the above-mentioned vocabulary for "mobilitydcatap:dataModel" ("DATEX II v2.3", "DATEX II v3.0" etc.).  However, this would make this vocabulary list quite complex, and some metadata users just want to know the model (only "DATEX II"), without the versioning. Thus, we want to keep the distinct (and optional) property "mobilitydcatap:dataFormatVersion".


However, to deal with potential mis-interpretations of  "mobilitydcatap:dataFormatVersion", we will refine this property as follows:
1. We will restrict the typing of the version via the usage notes , e.g. only allow short version identifiers without "v", "underscores" etc. 
2. We will rename the property to "mobilitydcatap:dataModelVersion", as we explicitly only want to tell the version of the data models (DATEX II etc), and not the versions of other format information (e.g., versions of the XML language). The usage note will be refined accordingly.
